src/directives/for-of/README.md

```diff
@@ -65,23 +65,34 @@ When the contents of the iterator changes, `igxForOf` makes the corresponding ch
 | igxForScrollContainer  | string          | Only the strings `vertical` and `horizontal` are valid and specify the scroll orientation                                  |
 | igxForContainerSize    | string          | The px-affixed size of the container along the axis of scrolling                                                           |
 | igxForScrollContainer  | IgxForOf | Optionally pass the parent `igxForOf` instance to create a virtual template scrolling both horizontally and vertically     |
-| igxForRemote | boolean | Enables remote virtualization. Should be used in combination with the onChunkLoading event, where the new data can be requested from a remote service. Note that the state.totalCount option of the igxFor directive should be updated with the actual total record count from the service. 
+| igxForRemote | boolean | Enables remote virtualization. Should be used in combination with the OnChunkPreload event, where the new data can be requested from a remote service. Note that the state.totalCount option of the igxFor directive should be updated with the actual total record count from the service. 
```

Comment:
@dkamburov "state.totalCount" is no longer valid. Also "total record count" -> "total item count".